books/rtl/rel11/lib/defs.lisp

```diff
@@ -141,10 +148,10 @@
     0))
 
 (defund exactp (x n)
+  (declare (xargs :guard (and (real/rationalp x) (integerp n))))
```

Comment:
@nadezhin This is a style thing, but do you mind changing the conjuncts of guards to be on different lines?  We'd just like it to be consistent.  Thanks again for this patch!